CMIP6Plus/TermsOfUse/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
+<html>
+<head>
+<title>CMIP6Plus Terms Of Use</title>
+<meta http-equiv="refresh" content="0; url=https://pcmdi.llnl.gov/CMIP6Plus/TermsOfUse/TermsOfUse6-5.html">
+</head>
+<p><a href="https://pcmdi.llnl.gov/CMIP6Plus/TermsOfUse/TermsOfUse6-5.html">Please click here to redirect to the correct site</a></p>
```

Comment:
Could we use relative links instead?
```suggestion
<meta http-equiv="refresh" content="0; url=TermsOfUse6-5.html">
</head>
<p><a href="TermsOfUse6-5.html">Please click here to redirect to the correct site</a></p>
```

Reply:
Yep, great idea. The intention here is that there is a persistent URL if this doc changes, so *6-6.html, *6-7.html etc